### ISSUE TITLE
[Issue]: musubi-trained framepack lora not accepted by sdnext

### ISSUE THREAD

**Issue by SAC020** [vladmandic/sdnext#4164]
### Issue Description

1. trained a i2v lora using musubi trainer
2. attempted to use it with sdnext, failed

lora file: https://drive.google.com/file/d/1avw_cry74PHys3F-Dhp5So1FLRl_wssR/view?usp=sharing

### Version Platform Description

22:38:54-186574 INFO     Starting SD.Next
22:38:54-189196 INFO     Logger: file="C:\ai\sdnext\sdnext.log" level=DEBUG host="SEB-PC" size=74 mode=create
22:38:54-191994 INFO     Python: version=3.11.9 platform=Windows bin="C:\ai\sdnext\venv\Scripts\python.exe"
                         venv="C:\ai\sdnext\venv"
22:38:54-584547 INFO     Version: app=sd.next updated=2025-08-27 hash=c41afe6b branch=dev
                         url=https://github.com/vladmandic/sdnext/tree/dev ui=dev
22:38:55-389121 INFO     Version: app=sd.next latest=2025-08-28T19:22:16Z hash=daae3ae9 branch=dev
22:38:55-405102 INFO     Platform: arch=AMD64 cpu=Intel64 Family 6 Model 165 Stepping 5, GenuineIntel system=Windows
                         release=Windows-10-10.0.26100-SP0 python=3.11.9 locale=('English_United States', '1250')
                         docker=False

### Relevant log output

```shell
22:40:31-925372 INFO     Startup time: total=116.17 checkpoint=55.06 torch=17.18 launch=9.09 numpy=8.91 installer=8.62
                         libraries=4.89 extensions=3.06 gradio=2.45 ui-extensions=1.37 bnb=0.60 ui-networks=0.57
                         ui-models=0.50 diffusers=0.38 detailer=0.36 cleanup=0.33 ui-control=0.31 ui-video=0.31
                         app-started=0.27 ui-caption=0.24 samplers=0.23 ui-txt2img=0.20 models=0.19 ui-img2img=0.18
                         ui-defaults=0.16 api=0.14 lora=0.11 onnx=0.10 upscalers=0.08 ui-info=0.06 ui-extras=0.06
                         ui-gallery=0.06
22:40:31-928902 DEBUG    Save: file="C:\ai\sdnext\config.json" json=29 bytes=1247 time=0.005
22:40:33-476799 DEBUG    FramePack load: module=encoder {'repo': 'lllyasviel/flux_redux_bfl', 'subfolder':
                         'feature_extractor'} model={'repo': 'lllyasviel/flux_redux_bfl', 'subfolder': 'image_encoder'}
Progress  5.99it/s ██████████████ 100% 1/1 00:00 00:00 Loading checkpoint shards
22:40:34-198900 DEBUG    FramePack load: module=transformer {'repo': 'lllyasviel/FramePackI2V_HY', 'subfolder': ''}
22:40:34-200133 DEBUG    Quantization: module="Model" type=sdnq mode=pre dtype=int6 matmul=True group_size=0
                         quant_conv=False matmul_conv=False dequantize_fp32=False quantize_with_gpu=True
                         quantization_device=cuda return_device=cpu device_map=default offload_mode=balanced
                         non_blocking=False modules_to_not_convert=[] modules_dtype_dict={}
Progress 1867.73it/s █████████████████████ 100% 3/3 00:00 00:00 Fetching 3 files
Progress 16.87s/it ██████████████ 100% 3/3 00:50 00:00 Loading checkpoint shards
22:41:25-906760 DEBUG    GC: current={'gpu': 1.6, 'ram': 22.8, 'oom': 0} prev={'gpu': 1.81, 'ram': 23.94} load={'gpu':
                         7, 'ram': 36} gc={'gpu': 0.21, 'py': 48191}
                         fn=postprocess_model:_process_model_after_weight_loading why=sdnq time=0.54
22:41:25-929216 INFO     FramePack load: model=FramepackHunyuanVideoPipeline variant="bi-directional" type=hunyuanvideo
                         time=97.04
22:41:25-930130 INFO     Offload: type=balanced op=init watermark=0.2-0.8 gpu=4.80-19.20:24.00 cpu=64.000 limit=0.00
                         always=['T5EncoderModel', 'UMT5EncoderModel'] never=['CLIPTextModel',
                         'CLIPTextModelWithProjection', 'AutoencoderKL']
22:41:25-952522 DEBUG    Module: name=transformer cls=HunyuanVideoTransformer3DModelPacked size=9.013 params=9660216384
                         quant=QuantizationMethod.SDNQ
22:41:25-957724 DEBUG    Module: name=text_encoder cls=LlamaModel size=5.854 params=5760356352
                         quant=QuantizationMethod.SDNQ
22:41:25-961588 DEBUG    Module: name=image_processor cls=SiglipVisionModel size=0.798 params=428225600 quant=None
22:41:25-964625 DEBUG    Module: name=vae cls=AutoencoderKLHunyuanVideo size=0.459 params=246478803 quant=None
22:41:25-966558 DEBUG    Module: name=text_encoder_2 cls=CLIPTextModel size=0.229 params=123060480 quant=None
22:41:25-967738 INFO     Model class=FramepackHunyuanVideoPipeline modules=5 size=16.354
22:41:26-194447 DEBUG    GC: current={'gpu': 1.6, 'ram': 22.8, 'oom': 0} prev={'gpu': 1.6, 'ram': 22.8} load={'gpu': 7,
                         'ram': 36} gc={'gpu': 0.0, 'py': 892} fn=load_model:load_model why=load time=0.23
22:41:26-227749 INFO     FramePack start: mode=i2v variant="bi-directional" frames=100 sections=3 resolution=640
                         seed=1070522063 duration=10 teacache=True thres=0.15 cfgzero=False
22:41:26-229175 INFO     FramePack params: steps=25 start=1 end=1 vision=1 scale=1 distilled=10 rescale=0 shift=3
22:41:26-250060 DEBUG    FramePack prepare: input="1024x1024" resized="608x640" resolution=640 scale=1.0
22:41:26-251892 DEBUG    Network load: type=LoRA name="musubi_4090_stock_framepack_lora-000015"
                         file="C:\ai\sdnext\models\Lora\musubi_4090_stock_framepack_lora-000015.safetensors" detected=
                         method=diffusers scale=1.0 fuse=False
22:41:26-938829 ERROR    Network load: type=LoRA name="musubi_4090_stock_framepack_lora-000015" adapters=[] not loaded
Video ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━  0/75   0% -:--:-- 0:00:04 vision encode22:41:31-696806 TRACE    FramePack prompt: system=153 user=75 optimized=True unmodified=False mode=optimized
Video ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━  0/75   0% -:--:-- 0:00:20 text encode section=1ptxas info    : 0 bytes gmem
```

### Backend

Diffusers

### Compute

nVidia CUDA

### Interface

Standard

### Branch

Dev

### Model

Any Video Model

### Acknowledgements

- [x] I have read the above and searched for existing issues
- [x] I confirm that this is classified correctly and its not an extension issue

**Comment by vladmandic** [comment]
i'll take a look, but honestly i haven't heard of musubi trainer - and unfortunately, there is no standard for loras, each trainer does something different (and in some cases, each training method of each trainer does things differently).
placing in backlog.

**Comment by SAC020** [comment]
I hear you. Would be great if you can do something about it - thanks!

This is the trainer repository: https://github.com/kohya-ss/musubi-tuner

This particular lora was trained with adamw8bit in fp8. I'll see if I can get a runpod to train it in bf16 without the fp8 "patch"

**Comment by kentrob-byte** [comment]
This might not be helpful since you are trying to do video loras, but I'm using musubi-tuner successfully to create t2i loras for Wan2.2. I use the following training command:

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 C:\musubi-tuner\src\musubi_tuner\wan_train_network.py --task t2v-14B --dit C:\musubi-tuner\models\diffusion_models\wan2.2_t2v_low_noise_14B_fp16.safetensors --dataset_config C:\musubi-tuner\dataset\dataset.toml --sdpa --mixed_precision fp16 --fp8_base --optimizer_type adamw8bit --learning_rate 3e-4 --gradient_checkpointing --gradient_accumulation_steps 1 --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --network_alpha 32 --timestep_sampling shift --discrete_flow_shift 1.0 --max_train_epochs 100 --save_every_n_epochs 20 --seed 666 --optimizer_args weight_decay=0.1 --max_grad_norm 0 --lr_scheduler polynomial --lr_scheduler_power 4 --lr_scheduler_min_lr_ratio="5e-5" --output_dir "C:\lora_out\Wan" --output_name "XYZ" --blocks_to_swap 20 --metadata_title XYZ--metadata_author XYZ

**Comment by SAC020** [comment]
Just for reference, this is a framepack i2v lora which works: https://drive.google.com/file/d/1TF4fkUkGX5sdka1pgeDcj71BMhnUyLUa/view?usp=sharing

Trained on civitai with musubi for hunyuan then converted with musubi for framepack. First thing I notice it's a larger file size, but can't tell anything more about differences.


kentrob: it's not just t2i vs i2v, but also wan vs framepack. But one thing that I notice is you have specific --network_alpha 32. I didn't specify it and musubi defaults to network_alpha 1 (??). But I don't think it would affect compatibility, just quality.

**Comment by SAC020** [comment]
This is a lora trained without the fp8 optimization, still doesn't load in sdnext: https://drive.google.com/file/d/12My-ZQmTnDdmYLoLykc9G-Hz-THgFAOr/view?usp=sharing

**Comment by vladmandic** [comment]
for compatibility, what matters is tool (musubi) + module (dit=wan2.2_t2v_low_noise_14B_fp16.safetensors )+ lora type (in this case `networks.lora_wan`).
other params have impact for training, but not compatibility.

**Comment by SAC020** [comment]
I don't know if it helps in any way adapting sdnext to load musubi framepack loras, but I was able to generate framepack videos with the musubi-trained lora following this documentation: https://github.com/kohya-ss/musubi-tuner/blob/main/docs/framepack.md . 

So I'm expecting that the code to load a musubi framepack lora is somewhere in that repository - again, I don't know if this helps or not to replicate the loading mechanism.

The lora worked as expected, but the video inference performance is **way** below the sdnext framepack implementation which is much faster.
